installer/conf/container.conf

```diff
@@ -64,3 +64,17 @@
 <filter oms.container.**>
 	type filter_container
 </filter>
+
+#separate buffer for containers
+<match oms.container.**>
+  type out_oms
+  log_level info
+  buffer_chunk_limit 20m
+  buffer_type file
+  buffer_path /var/opt/microsoft/omsagent/state/state/out_oms_docker*.buffer
```

Comment:
Removed the hardcoded path , Tested by modifying omsadmin.sh to update the buffer path during install. Image shows container.conf contents after onboarding to a workspace
![image](https://cloud.githubusercontent.com/assets/25937992/24307951/55e469b2-1083-11e7-97ce-65d501bb49ff.png)
Will send out a separate PR for the omsadmin.sh